projects/entities/projects/persistence/src/entity_base/transaction_table.rb

```diff
@@ -418,7 +418,7 @@ def find_many_by(attributes_filter)
 
         def find_all_by_attribute_containing_any_of(attribute_name, values)
           values = Util.array(values)
-          return [] if values.empty?
+          return Foobara::EMPTY_ARRAY if values.empty?
```

Comment:
Doesn't hurt at all, but I don't think we need the `Foobara::` prefix here since we're in that namespace. But I don't actually know which is more performant, if either is. So no need to change it. Just mentioning!